policies/stable-release-updates.md

```diff
@@ -6,8 +6,10 @@ This policy covers allowed changes on stable release branches.
 Definitions
 -----------
 
-A **stable release** is a patch release from an existing supported minor
-release branch.
+A **stable release** is a series including a major or minor release
+that is not a pre-release, and all its updates.
+
+A **patch release** is an update of a stable release.
```

Comment:
Hmm. The definition of "stable release" seems good now. But I'm not sure that "patch release" quite makes sense. Since the definition of "stable release" includes all the updates it doesn't make sense to say a patch release is an update of a stable release.

Possibly "A patch release is an update within a stable release series"?

Reply:
Done